File: src/contests/cg/puzzle/ruby/google-interview---the-two-egg-problem.rb
# 🎮 CodinGame Puzzle - google-interview---the-two-egg-problem
# https://www.codingame.com/training/hard/google-interview---the-two-egg-problem

n = gets.to_i
k = 0
k += 1 while k * (k + 1) / 2 < n
puts k

File: src/contests/cg/puzzle/vbnet/google-interview---the-two-egg-problem.vb
' 🎮 CodinGame Puzzle - google-interview---the-two-egg-problem
' https://www.codingame.com/training/hard/google-interview---the-two-egg-problem

Imports System

Module Solution
    Sub Main()
        Dim n As Long = Long.Parse(Console.ReadLine().Trim())
        Dim k As Long = 0
        While k * (k + 1) \ 2 < n
            k += 1
        End While
        Console.WriteLine(k)
    End Sub
End Module
